apps/ledger-live-mobile/src/families/casper/operationDetails.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { useTranslation } from "react-i18next";
+import Section from "../../screens/OperationDetails/Section";
+
+type Props = {
+  extra: {
```

Comment:
Same comment as previously done on the other part of the review, regarding upcoming https://github.com/LedgerHQ/ledger-live/pull/4235

Reply:
Fixed